humility-idol/src/lib.rs

```diff
@@ -285,6 +285,36 @@ fn lookup<'a>(
     }
 }
 
+// Convert string representation of array from the hiffy `--argument` list
+// to a vector. Arrays must be enclosed in square brackets with array
+// elements separated by the space character.
+// E.g.: '[37 1 255 127]'
```

Comment:
Nit: I'm on the fence whether we should support `[37, 1, 255, 127]`, since that's standard Rust syntax.  Perhaps we should just strip `,` from the string before processing it further, so that both flavors work?

Reply:
I went back and forth on this. I ended up using only the space character because it was the path of least resistance: the argument parsing code that the larger `--arguments` list expects arguments to be separated by commas. Having commas in the array string confuses the `--arguments` parser so there's additional work to make all of that play nice together. I'm happy to spend some cycles on getting to the bottom of this if you prefer.

Reply:
Gotcha, I don't feel strongly enough to volunteer you for a fight with `clap`'s argument parser.